playbooks/provisioning/openstack/galaxy-requirements.txt

```diff
@@ -0,0 +1,7 @@
+# This is the Ansible Galaxy requirements file to pull in the correct roles
+
+# FIXME(bogdando) uncomment, when it works
+
+# From 'infra-ansible'
+#src: https://github.com/redhat-cop/infra-ansible
+#version: master
```

Comment:
Can you rename the file to `galaxy-requirements.yaml` and change its contents to this?

    - src: https://github.com/redhat-cop/infra-ansible

That should solve the issue from https://github.com/redhat-cop/casl-ansible/pull/49